src/core/data/worker.c

```diff
@@ -93,12 +93,12 @@ static void
 worker_add_stream(void)
```

Comment:
Do you mind fixing the naming inconsistency _I_ introduced? we can call thish one `_worker_read_notification`, and the one used for return `_worker_write_notification`